integration-tests/helpers/start-server-with-environment.js

```diff
@@ -0,0 +1,22 @@
+const setupServer = require("./setup-server")
+const { initDb } = require("./use-db")
+
+const startServerWithEnvironment = async (cwd, env = {}) => {
+  Object.entries(env).forEach(([key, value]) => {
+    process.env[key] = value
+  })
+
+  const dbConnection = await initDb({
+    cwd,
+  })
+
+  const medusaProcess = await setupServer({
+    cwd,
+    verbose: true,
```

Comment:
todo: we would likely want to control this on a test-by-test basis.

suggestion: change startServerWithEnv to take an object:
```
startServerWithEnvironment({ cwd, env, verbose }) {
```